docs/network-spec/mux.tex

```diff
@@ -156,3 +171,5 @@ \section{Node-to-node and node-to-client protocol numbers}
   \caption{Node-to-client protocol numbers}
   \label{table:node-to-client-protocol-numbers}
 \end{table}
+
+\todo[inline]{Need subsection on how handshake is negotiated}
```

Comment:
What's the goal of this? To explain the protocol or the rules for the negotiation?